gems/activestorage/6.0/_test/Steepfile

```diff
@@ -8,11 +8,14 @@ target :test do
   library "pathname"
   library "logger"
   library "mutex_m"
+  library "cgi"
   library "date"
+  library "erb"
   library "monitor"
   library "singleton"
   library "tsort"
   library 'rack'
+  library 'thor'
```

Comment:
I noticed the test script for activestorage:6.0 has been broken. So I also updated this at the same time.